supervisor/jobs/decorator.py

```diff
@@ -181,6 +181,14 @@ def _check_conditions(self):
                 f"'{self._method.__qualname__}' blocked from execution, no Home Assistant OS-Agent available"
             )
 
+        if (
```

Comment:
Please add a test for this. It should be not too hard, you can test it without load and then after load (which work because of the dbus mockups)